x11-terms/zutty/zutty-0.11.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{7,8,9} )
+PYTHON_COMPAT=( python3_{8,9} )
```

Comment:
10? :)

Reply:
Just tested with py3.10 and seems to work. Looks like a few other waf ebuilds have moved too.

```
dev-libs/serd/serd-0.30.10.ebuild:PYTHON_COMPAT=( python3_{8..10} )
dev-libs/sord/sord-0.16.8-r1.ebuild:PYTHON_COMPAT=( python3_{8..10} )
dev-libs/sord/sord-9999.ebuild:PYTHON_COMPAT=( python3_{8..10} )
media-libs/lilv/lilv-0.24.12.ebuild:PYTHON_COMPAT=( python3_{8..10} )
media-libs/sratom/sratom-0.6.8.ebuild:PYTHON_COMPAT=( python3_{8..10} )
media-libs/suil/suil-0.10.10.ebuild:PYTHON_COMPAT=( python3_{8..10} )
media-sound/ardour/ardour-6.9.ebuild:PYTHON_COMPAT=( python3_{7,8,9,10} )
media-sound/ardour/ardour-9999.ebuild:PYTHON_COMPAT=( python3_{7,8,9,10} )
media-sound/guitarix/guitarix-0.43.1.ebuild:PYTHON_COMPAT=( python3_{7..10} )
media-sound/guitarix/guitarix-9999.ebuild:PYTHON_COMPAT=( python3_{7..10} )
media-sound/jack2/jack2-1.9.19.ebuild:PYTHON_COMPAT=( python3_{8,9,10} )
media-video/mpv/mpv-0.34.0.ebuild:PYTHON_COMPAT=( python3_{8..10} )
media-video/mpv/mpv-9999.ebuild:PYTHON_COMPAT=( python3_{8..10} )
net-fs/samba/samba-4.15.2-r2.ebuild:PYTHON_COMPAT=( python3_{8..10} )
net-fs/samba/samba-4.15.3-r1.ebuild:PYTHON_COMPAT=( python3_{8..10} )
sys-libs/ldb/ldb-2.4.1.ebuild:PYTHON_COMPAT=( python3_{8..10} )
```